### DISCUSSION TITLE
posts/Go/deploy_go_app/

### DISCUSSION THREAD

**Discussion by Q1mi** [Q1mi/BlogComments#328] [Announcements]
https://www.liwenzhou.com/posts/Go/deploy_go_app/ 

 李文周的Blog gin go 项目部署 web nginx supervisor nohup 前后端分离 cors 跨域 static vue index

**Comment by TeslaLyon** [comment]
您好，很感谢您的发表，请问此文中的流程图是用什么软件绘制的？😬

**Comment by chenweil** [comment]
感谢小王子，按照Supervisor内容，部署了自己的程序。

**Comment by 471909488** [comment]
aaaaaaa

**Comment by mant7s** [comment]
老师，什么时候出一篇基于k8s部署go项目的文章
